WinUIGallery/Package.WAP.Dev.appxmanifest

```diff
@@ -0,0 +1,53 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Package
+    xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+    xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+    xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+    xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3"
+    xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+    IgnorableNamespaces="uap mp uap3">
+  <Identity Name="Microsoft.WinUI3ControlsGallery.Debug" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="1.3.13.0" />
+  <mp:PhoneIdentity PhoneProductId="0ba9d13a-a477-463f-955f-91d935fa4cd5" PhonePublisherId="00000000-0000-0000-0000-000000000000" />
+  <Properties>
+    <DisplayName>WinUI 3 Gallery Dev</DisplayName>
+    <PublisherDisplayName>Microsoft Corporation</PublisherDisplayName>
+    <Logo>Assets\Tiles\StoreLogo-sdk.png</Logo>
+    <uap:SupportedUsers>multiple</uap:SupportedUsers>
+  </Properties>
+  <Dependencies>
+    <TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.17134.0" MaxVersionTested="10.0.18362.0" />
+  </Dependencies>
+  <Resources>
+    <Resource Language="x-generate" />
+  </Resources>
+  <Applications>
+    <Application Id="App" Executable="$targetnametoken$.exe" EntryPoint="$targetentrypoint$">
+      <uap:VisualElements DisplayName="WinUI 3 Gallery Dev" Square150x150Logo="Assets\Tiles\squaretile-sdk.png" Square44x44Logo="Assets\Tiles\SmallTile-sdk.png" Description="WinUI 3 Gallery Debug" BackgroundColor="transparent">
```

Comment:
Huh, I was able to build from this branch, but I didn't see the 'Dev' title reflect on the running app, was going to come and suggest this, as having a visual distinction is really useful (we did this in the Toolkit as well):

![image](https://user-images.githubusercontent.com/24302614/174900911-bc7c74f1-8c83-4d86-bf03-cb2c2a587f59.png)

Reply:
Ah, I see it in the start menu, but that's not helpful if you run the apps side-by-side and are trying to compare (and if you're not running in debug mode or turn the XAML toolbar off, you can lose track of which is which).

![image](https://user-images.githubusercontent.com/24302614/174901137-95123985-d73a-4a87-9bab-e8361c13aae5.png)

Reply:
Very good paint, to be honest forgot a bit about that. But now it should be also display it in the title of the app.